examples/zcash-deposit.ts

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env node
+
+/**
+ * Zcash Deposit Example
+ *
+ * Two-step process to deposit Zcash and receive nZEC on NEAR:
+ * 1. Generate deposit address → Send Zcash to it
+ * 2. Finalize deposit after Zcash confirmation
+ *
+ * Setup:
+ * 1. Replace NEAR_ACCOUNT with your testnet account
+ * 2. Replace TX_HASH and VOUT with your Zcash transaction details
+ * 3. Ensure NEAR credentials are in ~/.near-credentials
+ *
+ * Usage: bun run examples/bitcoin-deposit.ts
+ */
+
+import os from "node:os"
+import path from "node:path"
+import { Account } from "@near-js/accounts"
+import { getSignerFromKeystore } from "@near-js/client"
+import { UnencryptedFileSystemKeyStore } from "@near-js/keystores-node"
+import { JsonRpcProvider } from "@near-js/providers"
+import { NearBridgeClient } from "../src/clients/near.js"
+import { setNetwork } from "../src/config.js"
+
+// Configuration - Replace with your values
+const NEAR_ACCOUNT = "bridge-sdk-test.testnet"
+const NETWORK = "testnet" as "testnet" | "mainnet"
+
+// Step 2 configuration - Add these after sending Zcash
+const TX_HASH = "cef976d783ac338d5d7bca47cd054f372deb5dc53dc70960321c12b8048292f4" // Your Zcash transaction hash
+const VOUT = 0 // Output index (usually 0 or 1)
+
+setNetwork(NETWORK)
+
+async function main() {
+  console.log("🚀 Zcash Deposit Example")
+
+  // Initialize NEAR client
+  const keyStore = new UnencryptedFileSystemKeyStore(path.join(os.homedir(), ".near-credentials"))
+  const signer = await getSignerFromKeystore(NEAR_ACCOUNT, NETWORK, keyStore)
+  const provider = new JsonRpcProvider({
+    url: "https://rpc.testnet.near.org",
+  })
+  const account = new Account(NEAR_ACCOUNT, provider, signer)
+
+  const bridgeClient = new NearBridgeClient(account, "omni.n-bridge.testnet")
+
+  // Get minimum deposit amount
+  const config = await bridgeClient.getZcashBridgeConfig()
+  console.log(`Minimum deposit: ${config.min_deposit_amount} zatoshis`)
```

Comment:
Gas is not taken into account here. Therefore, you need to transfer min_deposit_amount + gas_fee.